scheduler/src/cook/kubernetes/api.clj

```diff
@@ -314,12 +316,12 @@
 
         (if (str/blank? continue)
           (do
-            (log/info "In" compute-cluster-name
-                      "compute cluster, done listing pods for all namespaces"
-                      {:limit limit
-                       :number-pods (count new-all-pods)
-                       :number-pods-this-chunk (count pods)
-                       :resource-version resource-version})
+            (log-structured/info "Done listing pods for all namespaces"
+                                 {:compute-cluster compute-cluster-name
+                                  :limit limit
+                                  :number-pods (count new-all-pods)
+                                  :number-pods-this-chunk (count pods)
+                                  :resource-version resource-version})
```

Comment:
Does this effectively absorb the existing logged clojure dict into the metadata dict for structured logging? Are we worried about cardinality then?

Reply:
It does absorb the dictionary, but I don’t think we need to worry about the cardinality here since it’s a small number of fixed fields. 

There were a few places where the dict data would end up adding a large number of varying fields, and for those I converted it to a string explicitly so it is all logged as one json blob under one field. 

But in general a few tens of fields across the different messages should not be a problem for log field cardinality.